### DISCUSSION TITLE
A request

### DISCUSSION THREAD

**Discussion by vichusachu** [sayaka-sh/spmp#220] [General]
Sir,can u please provide me an apk without any restrictions i mean i have to edit layout , change icons etc of my taste,i dont want to make any piracy things or something so if u can please provide me one apk of latest version,btw ur work is really awesome❣️

**Comment by sayaka-sh** [comment]
Obfuscation is already disabled afaik. If you'd point me to any files or documentation showing what I need to change, I'd be happy to take a look.

Otherwise, you're more than welcome to fork  the project and compile it yourself.
